.github/workflows/ci-nightly.yml

```diff
@@ -0,0 +1,42 @@
+name: CI - FAST Publish
+
+on:
+  pull_request:
+    branches:
+    - master
+  # schedule:
+  # - cron: 0 12 * * 7
```

Comment:
In production, this will trigger in GitHub Actions on `push`.  Until testing is complete, this will trigger on each `pull_request`.  Final commit will reconcile this.